src/renderer/containers/Header/UserMenu.tsx

```diff
@@ -88,25 +108,38 @@ export const UserMenu: React.FC<{
           Logout
         </MenuItem>
       </Menu>
-      <Dialog
-        fullScreen={fullScreen}
+      <ConfirmationModal
+        title="Change your Display Name"
+        open={openNameChangePrompt}
+        onClose={handleClose}
+        onSubmit={onChangeName}
+        fullWidth={false}
+      >
+        <Typography paragraph={true}>Enter the new name in the field below</Typography>
+        <TextField
+          required={true}
+          id="displayName"
+          variant="outlined"
```

Comment:
I don't think we've used outlined inputs very much in the rest of the app forms. Maybe we should stick with "filled"?

```suggestion
          variant="filled"
```